### PR TITLE
Add open graph tags

### PR DESCRIPTION
**Pull request by adammacias**
References:
- [The Open Graph protocol](http://ogp.me/)
- [Open Graph Stories Facebook](https://developers.facebook.com/docs/sharing/opengraph)
- [Twitter Cards and Open Graph](https://dev.twitter.com/cards/getting-started)

### CODE REVIEW

**Comment by adammacias** [comment]
:up: @mdo @cvrebert

File: _includes/header.html [hunk 1]
```diff
@@ -24,6 +24,20 @@
...
+    <meta content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}" property="og:description">
+    <meta content="{{ site.url }}{% if page.url %}{{ page.url }}{% endif %}" property="og:url">
+    <meta content="{{ site.name }}" property="og:site_name">
+    <meta content="{{ site.url }}/img/ogp.png" property="og:image">
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:image:width" content="1200">
+    <meta property="og:image:height" content="630">
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@getbootstrap" />
+    <meta name="twitter:creator" content="@getbootstrap" />
```
Comment (mdo) [inline _includes/header.html]: Some of these are self-closing—with the trailing space and slash—and others are not. Perhaps nuke all the trailing stuff?

**Comment by adammacias** [comment]
Done @mdo :relaxed:

**Comment by mdo** [comment]
Cool I'll test these out in a bit and get back to you soon :).

**Comment by adammacias** [comment]
Perfect! I'll be waiting ;)

**Comment by adammacias** [comment]
@mdo Did you test that? :dancers:

**Comment by mdo** [comment]
Looks fine to me! Thanks!

**Pull request merged**